decisions/employment_decisions.txt

```diff
@@ -630,7 +630,13 @@ decisions = {
 				save_event_target_as = invited_character
 				
 				# Warcraft
-				set_prev_gfx_effect = yes
+				if = {
+					limit = { ROOT = { NOT = {  trait = creature_ancient_coral } } }
+					set_prev_gfx_effect = yes
+				}
+				else = {
+					set_graphical_culture = ancient_coral
+				}
```

Comment:
Maybe you just allow setting gfx in `set_prev_gfx_effect`. If you play a female, you can't marry another female.
Though it will allow getting characters with really high stats for the players so maybe there's a better way.

Reply:
female problem no longer exists since we have present bachelor. i could add cooldown for those decisions for coral ancients since theyre rare and theres not many of them

Reply:
> female problem no longer exists since we have present bachelor. i could add cooldown for those decisions for coral ancients since theyre rare and theres not many of them

I guess the cooldown will be nice, yeah.